### PR TITLE
Switch ppc64le builds in the Arch rebuild to Azure

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

**Comment by isuruf** [comment]
I think we should stop the arch migrator as reverting this is not going to be easy.

**Comment by CJ-Wright** [comment]
We could change the default on smithy.

**Comment by beckermr** [comment]
I'd be fine with either solution. I don't see how this is hard to undo. If ppl have trouble with the emulated builds, they can change them themselves later.

**Comment by CJ-Wright** [comment]
Or if we decide to go back to default we write a mini migratior for it and let the bot work it's way through it.

**Comment by isuruf** [comment]
> Or if we decide to go back to default we write a mini migratior for it and let the bot work it's way through it.

What happens to feedstocks that genuinely need `azure`?

**Comment by CJ-Wright** [comment]
This is why I prefer setting the default to azure. For those R packages that have gotten moved we can move them back, we have the needed data.

**Comment by xhochy** [comment]
I would prefer the smithy solution for now as this would move all packages to Azure for now but once they get build again, they could then run on the then-preferred solution (which could be neither Travis nor Azure depending on what we get to).

**Comment by beckermr** [comment]
We are putting in azure by hand for R 4. Should we take that out too I guess?

File: conda_forge_tick/migrators/arch.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class ArchRebuild(GraphMigrator):
     }
     arches = {
         "linux_aarch64": "default",
-        "linux_ppc64le": "default",
+        "linux_ppc64le": "azure",
```
Comment (isuruf) [inline conda_forge_tick/migrators/arch.py:36]:
```suggestion
        "linux_ppc64le": "emulated",
```

**Comment by isuruf** [comment]
Running emulated builds is fine for R packages, but switching the default would be problematic for other packages taking more than 30 minutes as they would easily take more than 6 hours to complete using QEMU.

I've suggested a different strategy above.

**Comment by beckermr** [comment]
Can you explain your suggestion? I am legit confused...

**Comment by isuruf** [comment]
`emulated` is another synonym for `azure`, but we can switch this back easily because this is not used by any feedstock atm.

**Comment by beckermr** [comment]
This is a good idea, but maybe we should add a special synonym for this purpose only? Hacking over emulated is not great since that word means emulated and then it might not.

**Comment by scopatz** [comment]
Is there a status update on this?

**Comment by beckermr** [comment]
Travis is still flaky but better. I’d say we are kind of in limbo.

**Comment by CJ-Wright** [comment]
Have we resolved the R issues? I remember there being some question about if azure could produce valid R builds for non x86 chipsets

**Comment by CJ-Wright** [comment]
also xref: https://github.com/conda-forge/conda-smithy/issues/1309

**Comment by beckermr** [comment]
AFAIK we have not resolved the R 4 issue.

**Comment by CJ-Wright** [comment]
I think this is mostly moot, since this is controlled in conda-smithy now.

**Pull request closed without merging**